haystack/management/commands/update_index.py

```diff
@@ -248,6 +255,12 @@ def handle(self, **options):
         elif self.verbosity > 1:
             LOG.setLevel(logging.INFO)
 
+        if (minutes and age) or (minutes and start_date) or (age and start_date):
+            raise NotImplementedError("Minutes / age / start date options are mutually exclusive")
+
+        if minutes is not None:
+            self.start_date = now() - timedelta(minutes=int(minutes))
```

Comment:
Since the argument is defined as an integer, the extra `int()` call probably isn't necessary.

Reply:
ah yeah was just consistency with age but i suppose we could dump both